linera-storage/src/scylla_db.rs

```diff
@@ -59,7 +60,7 @@ impl ScyllaDbStore {
 pub type ScyllaDbStoreClient<C> = DbStoreClient<ScyllaDbClient, C>;
 
 #[cfg(any(test, feature = "test"))]
-impl ScyllaDbStoreClient<crate::TestClock> {
```

Comment:
Is this a fix or just a cosmetic choice here?

Reply:
It was both: a fix and cosmetic.